src/Components/UI/Input/InputTypeOne.jsx

```diff
@@ -0,0 +1,22 @@
+const Input = (props) => {
+  return (
+    <div className={props.inputWrapper}>
+      <label>
+        {props.label}
+        <div className={props.iconWrapper}>
+          <i className={props.icon}></i>
+          <input
+            type={props.type}
+            name={props.name}
+            required={props.required}
+            autoComplete={props.autoComplete}
+            placeholder={props.placeholder}
+            onChange={props.onChange}
+          />
```

Comment:
Suggestion: You could destructure props here to make the code clean,

example: const { inputWrapper, iconWrapper } = props;

Reply:
Noted will make these changes.